layouts/partials/header.html

```diff
@@ -34,13 +34,6 @@
   {{ $styles := resources.Get "css/docs.css" | fingerprint }}
 
   <link rel="stylesheet" href="{{ $styles.Permalink }}" integrity="{{ $styles.Data.Integrity }}">
```

Comment:
Last changed in themes in Nov 30, 2022:
https://github.com/camunda/camunda-docs-theme/commit/1f9e19d10967dc831d56775954dbb46a193135a0#diff-ef4bb046d34a9de9dcbb20a75fc9ced0448446808348683926f22bd7192c654aL6

Section removed from docs-static: Aug 8, 2023
https://github.com/camunda/camunda-docs-static/commit/0b71c15dfb4af07c230c17b903dcfe1659c6d0f6#diff-e76d120cd614de8045567875ede423e24392c156e53da5b954615898f4b2d54bL38

For context see static PR:https://github.com/camunda/camunda-docs-static/pull/509#discussion_r1830938770

Reply:
did not change `<nav>` on line 110 as it would reintroduce this bug: https://github.com/camunda/camunda-bpm-platform/issues/3721
(tested locally)